graphql_compiler/schema_generation/sqlalchemy/utils.py

```diff
@@ -4,15 +4,22 @@
 
 def validate_that_tables_have_primary_keys(tables):
     """Validate that each SQLAlchemy Table object has a primary key."""
+    tables_missing_primary_keys = set()
```

Comment:
We are currently in the process of type-hinting the entire compiler codebase, so our general rule of thumb is that if you edit a function, since you already have to understand what it does, you should also add type hints to it — including arguments, return values, and any intermediate values. For example, `tables_missing_primary_keys = set()` could be `tables_missing_primary_keys: Set[Table] = set()` or something along those lines.